spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -26,8 +27,11 @@
 from spyder.api.translations import _
 from spyder.config.base import is_conda_based_app
 from spyder.plugins.updatemanager.workers import (
+    UPDATER_PATH,
+    UpdateType,
     validate_download,
     WorkerUpdate,
+    WorkerUpdateUpdater,
     WorkerDownloadInstaller
 )
 from spyder.utils.conda import find_conda, is_anaconda_pkg
```

Comment:
```suggestion
from spyder.utils.conda import find_conda, is_anaconda_pkg
from spyder.utils.palette import SpyderPalette
```

To pass to Spyder-updater the color for icons (see below).